tox.ini

```diff
@@ -4,9 +4,11 @@ envlist = py27,py3,dandruff
 [testenv]
 use_develop=True
 deps=-r{toxinidir}/requirements.txt
-commands=py.test
+commands=
```

Comment:
@deryckh I think this is a fine example for why we should just use tox in travis now. Doesn't make a lot of sense for travis to have one way of running tests, and for us to have another.